rt/features/logging/src/main/java/org/apache/cxf/ext/logging/AbstractLoggingInterceptor.java

```diff
@@ -102,8 +110,12 @@ public void createExchangeId(Message message) {
         }
     }
 
-    protected String transform(final String originalLogString) {
+    protected String transform(final Message message, final String originalLogString) {
         return originalLogString;
     }
 
+    protected String maskSensitiveElements(final Message message, String originalLogString) {
+        return (new MaskSensitiveHelper(sensitiveElementNames))
```

Comment:
MaskSensitiveHelper is thread-safe, so it would be more performant only to create it once when sensitiveElementNames are set?

Reply:
yes, it is make sense

Reply:
only the thing that can contradict: I would like to add sensitive http headers names as well, if new separate setter for headers names will be introduced, instantiation of helper will be weird

Reply:
But perhaps it doesn't make sense to introduce separate setters for elements and headers, user can put them in the same list

Reply:
Ignore my last two comments - headers must be masked in different code and shouldn't be initialized in MaskSensitiveHelper at all